packages/e2e-test-app/test/ViewComponentTest.test.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ *
+ * @format
+ */
+
+import { goToComponentExample, dumpVisualTree } from './framework';
+
+beforeAll(async () => {
+    await goToComponentExample('View');
+});
+
+describe('ViewComponentTest', () => {
+    /* Test case #1: display:none disabled */
+    test('ViewComponentTestWithoutDisplayNone', async () => {
+        const dump = await dumpVisualTree('view-component-switch-view');
+        expect(dump).toMatchSnapshot();
+    });
+
+    /* Test case #2: Click button once, update view*/
+    test('ViewComponentTestWithDisplayNone', async () => {
+        await toggleDisplayNone();
+        const dump = await dumpVisualTree('view-component-switch-view');
+        expect(dump).toMatchSnapshot();
+    });
+
+    /* Test case #3: Click button one more, return to #1*/
```

Comment:
don't think we need this case - it basically tests if the Switch works. the first 2 tests cover both cases of visibility already. Also, each of these is a separate test, so you wouldn't 'return' to a previous state because there is none